content/blogs/2023-11-15-kestra-sqs.md

```diff
@@ -0,0 +1,193 @@
+
+Our flow will separate the two message publishing tasks with a simple Python script task that sleeps for a couple of seconds:
+
+```yaml
+id: publish-to-sqs
+namespace: dev
+
+tasks:
+  - id: sqsMessageStart
+    type: io.kestra.plugin.aws.sqs.Publish
+    from:
+      data: "FLOW: publish-to-sqs | TASK ID: sqsMessageStart | Flow started."
+
+  - id: pythonScript
+    type: io.kestra.plugin.scripts.python.Script
+    warningOnStdErr: false
+    script: |
+      import time
+
+      time.sleep(5)
+
+  - id: sqsMessageEnd
+    type: io.kestra.plugin.aws.sqs.Publish
+    from:
+      data: "FLOW: publish-to-sqs | TASK ID: sqsMessageEnd | Flow finished."
+
+
+taskDefaults:
+  - type: io.kestra.plugin.aws.sqs.Publish
+    values:
+      accessKeyId: "{{ secret('AWS_ACCESS_KEY') }}"
+      secretKeyId: "{{ secret('AWS_SECRET_KEY') }}"
+      region: "{{ secret('AWS_REGION') }}"
+      queueUrl: "{{ secret('SQS_URL') }}"
+```
+
+This is what your editor should look like:
+
+![Image 5 - Kestra editor contents](/blogs/2023-11-15-kestra-sqs/5.png)
+
+You can **run the flow** by clicking on the "Execute" button in the top right corner. You'll see the following Gantt view immediately afterward:
+
+![Image 6 - Flow execution Gantt view](/blogs/2023-11-15-kestra-sqs/6.png)
+
+The good thing about SQS is that it allows us to **see how many messages are currently in the queue**, straight from the AWS site:
+
+![Image 7 - Flow execution Gantt view](/blogs/2023-11-15-kestra-sqs/7.png)
+
+Long story short, it's safe to assume the messages were successfully published from Kestra to SQS.
+
+Let's see how to read them next.
+
+
+
+### Example 2: Process Messages from AWS SQS
+We've used the `io.kestra.plugin.aws.sqs.Publish` plugin to send messages to SQS, so can you guess the name of the plugin used to do the opposite? If you've guessed `io.kestra.plugin.aws.sqs.Consume`, you're absolutely right!
+
+This plugin requires you to set either the `maxDuration` or `maxRecords` parameter. Since we already know there are 2 messages in a queue, we'll use the latter, just for convenience. Once again, the credentials will be stored in `taskDefaults` for this specific plugin.
+
+There's not much to it - the plugin is straightforward to use, but there are a couple of [extra parameters available](https://kestra.io/plugins/plugin-aws/tasks/sqs/io.kestra.plugin.aws.sqs.consume) if you want to dig deeper:
+
+```yaml
+id: read-from-sqs
+namespace: dev
+
+tasks:
+  - id: consumeSqsMessages
+    type: io.kestra.plugin.aws.sqs.Consume
+    maxRecords: 2
+
+
+taskDefaults:
+  - type: io.kestra.plugin.aws.sqs.Consume
+    values:
+      accessKeyId: "{{ secret('AWS_ACCESS_KEY') }}"
+      secretKeyId: "{{ secret('AWS_SECRET_KEY') }}"
+      region: "{{ secret('AWS_REGION') }}"
+      queueUrl: "{{ secret('SQS_URL') }}"
+```
+
+This is what your flow editor should look like:
+
+![Image 8 - Kestra editor contents](/blogs/2023-11-15-kestra-sqs/8.png)
```

Comment:
```suggestion
```

This makes the post unnecessarily long. if you add a code block, no need to add a screenshot showing the code in the Editor.